src/Shared/KnownUrls.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting;
+
+internal static class KnownUrls
+{
+    public static class DataExplorer
```

Comment:
This is the most .NET thing ever. Why is this shared it's cosmos specific.

Reply:
Cos localization has to happen in the context of the dashboard. We do this for known commands too. We can add stuff to this for any other known link text we want to localize.